src/libraries/System.IO.Ports/Directory.Build.props

```diff
@@ -4,6 +4,7 @@
     <StrongNameKeyId>Open</StrongNameKeyId>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
     <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
+    <!-- This should be replaced with servicing version property in next iteration.-->
```

Comment:
We shipped 5.0.1 with the same assembly version because there were any code changes.
In the next release we will jump the assembly version directly to 5.0.0.2 for consistency .

Reply:
I see.  We don't want to do it now as it would invalidate P2P (unless we shipped the new one).
Nit: consider the following for clarity:
```xml
<!-- Replace <PackageVersion>5.0.1</PackageVersion> with
<ServicingVersion>2</ServicingVersion> when servicing this assembly. -->
```